flexmeasures/api/v3_0/users.py

```diff
@@ -89,7 +91,25 @@ def index(self, account: Account, include_inactive: bool = False):
         :status 403: INVALID_SENDER
         :status 422: UNPROCESSABLE_ENTITY
         """
-        users = get_users(account_name=account.name, only_active=not include_inactive)
+
+        if account is not None:
```

Comment:
Couldn't the code i0n here which deals with getting the accounts be similar to the code currently in the API endpoint for accounts?

It seems that one has two features we could also use here (take the current user's account if None passed, and maybe we want to raise if the user doesn't have access to the account they sent in.

Maybe worth a small utility function.